README.md

```diff
@@ -1,13 +1,11 @@
 # Project Todos
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is a Todo-app in which the user can type down tasks and then check/uncheck them and remove them from the list. The app is built with, React and Redux.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I faced some problems with keeping the app from rendering when pressing the refresh site button. I'm currently looking into how to handle that. 
```

Comment:
You mean building the local storage to prevent the added tasks from disappearing after refreshing the page?